flexget/plugins/filter/imdb.py

```diff
@@ -83,37 +84,68 @@ class FilterImdb(object):
-            'reject_languages': {'type': 'array', 'items': {'type': 'string'}},
-            'accept_languages': {'type': 'array', 'items': {'type': 'string'}},
-            'reject_actors': {'type': 'array', 'items': {'type': 'string'}},
-            'accept_actors': {'type': 'array', 'items': {'type': 'string'}},
-            'reject_directors': {'type': 'array', 'items': {'type': 'string'}},
-            'accept_directors': {'type': 'array', 'items': {'type': 'string'}},
-            'reject_writers': {'type': 'array', 'items': {'type': 'string'}},
-            'accept_writers': {'type': 'array', 'items': {'type': 'string'}},
-            'reject_mpaa_ratings': {'type': 'array', 'items': {'type': 'string'}},
-            'accept_mpaa_ratings': {'type': 'array', 'items': {'type': 'string'}}
+            'accept': {'$ref': '#/definitions/imdb_rule_list'},
+            'reject': {'$ref': '#/definitions/imdb_rule_list'},
+            'accept_excluding': {'$ref': '#/definitions/imdb_rule_list'},
+            'reject_excluding': {'$ref': '#/definitions/imdb_rule_list'},
+            'rest': {'type': 'string', 'enum': ['accept', 'reject']},
         },
-        'additionalProperties': False
+        'additionalProperties': False,
+        'definitions': {
+            'imdb_rule_list': {
+                'type': 'array',
+                'items': {
+                    'min_year': {'type': 'integer'},
+                    'max_year': {'type': 'integer'},
+                    'min_votes': {'type': 'integer'},
+                    'min_score': {'type': 'number'},
+                    'genres': {'type': 'array', 'items': {'type': 'string'}},
+                    'languages': {'type': 'array', 'items': {'type': 'string'}},
+                    'actors': {'type': 'array', 'items': {'type': 'string'}},
+                    'directors': {'type': 'array', 'items': {'type': 'string'}},
+                    'writers': {'type': 'array', 'items': {'type': 'string'}},
+                    'mpaa_ratings': {'type': 'array', 'items': {'type': 'string'}}
+                }
+            }
+        }
     }
 
     # Run later to avoid unnecessary lookups
     @plugin.priority(120)
     def on_task_filter(self, task, config):
-
+        rest = []
+        for operation, imdb_rules in config.items():
+            if operation == 'rest':
+                continue
+            leftovers = self.filter(task, operation, imdb_rules)
+            if not rest:
+                rest = leftovers
+            else:
+                # If there is already something in rest, take the intersection with r (entries no operations matched)
+                rest = [entry for entry in leftovers if entry in rest]
+
+        if 'rest' in config:
+            rest_method = Entry.accept if config['rest'] == 'accept' else Entry.reject
+            for entry in rest:
+                log.debug('Rest method %s for %s' % (config['rest'], entry['title']))
```

Comment:
let the logger do its own string interpolation:
`log.debug('Rest method %s for %s', config['rest'], entry['title'])`